src/Sound.ts

```diff
@@ -1,22 +1,38 @@
+import { generateId } from "./util/id";
+
+type SoundDataFormat = "mp3" | "wav" | "wave";
+
 export default class Sound {
   public name: string;
+  public id: string;
+
+  public dataFormat: SoundDataFormat;
+  public data: any;
+
   public md5: string;
   public ext: string;
+
   public sampleCount: number;
   public sampleRate: number;
-  public format: string;
-  public asset: any;
 
   constructor(options: {
     name: string;
+    id?: string;
+
+    dataFormat: SoundDataFormat;
```

Comment:
What's the difference between `dataFormat` and `ext`?

Reply:
I don't think so; AFAIK, there's no difference. Worth removing from Costume too. Thanks!

Reply:
(I guess my pending comment here disappeared...?) We don't need both - I got rid of ext in Costume too. Thanks!